usaspending_api/search/v2/views/spending_by_award_count.py

```diff
@@ -150,3 +151,26 @@ def query_elasticsearch_for_prime_awards(self, filters) -> list:
             "other": other,
         }
         return response
+
+    def query_elasticsearch_for_subawards(self, filters) -> list:
+        query_with_filters = QueryWithFilters(QueryType.SUBAWARDS)
+        filter_query = query_with_filters.generate_elasticsearch_query(filters)
+        s = SubawardSearch().filter(filter_query)
```

Comment:
This looks right for the most part. The only piece from the previous SQL that is missing from your ES query is the `WHERE award_id IS NOT NULL` filter that was previously present. You only want to return subawards where the `award_id` field exists (is not none/null).

```suggestion
        s = SubawardSearch().filter(filter_query).filter("exists", field="award_id")
```